docs/adr/XXXX-use-poetry-for-python-packaging-and-dependency-management.md

```diff
@@ -0,0 +1,65 @@
+# Use Poetry for Python packaging and dependency management
+
+* Status: proposed
+* Date: 2020-04-07
+
+## Context and Problem Statement
+
+Standard `setuptool` forces up to use 5 different files to create a package, making it
+quite complicated to create packages. we want to find an alternative.
+
+## Considered Options
```

Comment:
Official Python guidelines : https://packaging.python.org/guides/tool-recommendations recommend also to use `pip`. It could be listed in considered options.